CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## vNext (TBD)
 
+### Breaking changes
+* The generated parts are now named `.realm.dart` instead of `.g.dart`. This is because the builder is now a `PartBuilder`, instead of a `SharedPartBuilder`. This implies that other builders can consume the output.
+
```

Comment:
That would be a great improvement. 
The thing that is not clear for me is how the users will use the other builders. Do they have to add their attributes each time the `RealmObject` is generated? Can they put the attributes on the private class `_Person` ? Will we copy all of them to the output class `Person` during the generating of the model? How we will keep their attributes for the other builders?

Reply:
That is separate to this PR. One way I have been thinking about is to lift annotations from the model class to the generated realm object class.